### DISCUSSION TITLE
What does this mean, will there be a charge after 9.0？？？

### DISCUSSION THREAD

**Discussion by wz172** [PrismLibrary/Prism#3155] [General] [answered]
Licensing
The Prism Team would first and foremost like to thank all of those developers who have stepped up over the past 4 years with GitHub Sponsors. We are committed to ensuring the longevity and success of the Prism Library. As a result Prism 9.0 is now [Dual License](https://github.com/PrismLibrary/Prism/blob/master/LICENSE). We continue to offer a FREE Community License for the vast majority of our community, while the Commercial License will now be required by larger organizations to help fund and support the development of Prism. We additionally have the Commercial Plus License which grants access to a number of additional support libraries that build on top of Prism as well as a private Discord channel where you can ask questions and interact with the Prism Team

**Comment by dansiegel** [comment, marked as answer]
If you are using 9.0 and you do not qualify for the Community License then you must either purchase the Commercial or the Commercial Plus license.
